forc-util/src/lib.rs

```diff
@@ -5,16 +5,17 @@ use annotate_snippets::{
 };
 use anyhow::{bail, Context, Result};
 use forc_tracing::{println_action_green, println_error, println_red_err, println_yellow_err};
+use fuels_core::{codec::ABIDecoder, types::param_types::ParamType};
```

Comment:
This is causing the cyclic dependency :) for more details you can see the other comment